dotnet/test/Microsoft.AutoGen.Integration.Tests/HelloAppHostIntegrationTests.cs

```diff
@@ -43,7 +43,7 @@ public async Task AppHostLogsHelloAgentE2E(TestEndpoints testEndpoints)
             }
         }
         //sleep 5 seconds to make sure the app is running
-        await Task.Delay(15000);
+        await Task.Delay(20000);
```

Comment:
This seems increasing. Is there a better way to handle the "make sure the app is running"?
Also inconsistent with comment.

Reply:
ahh good catch on the comment - fixed that. Yes it's brittle.I had to increase it because @LittleLittleCloud needs a faster laptop ;-). Issue opened for it: https://github.com/microsoft/autogen/issues/4697